Workbooks/Failures/New Failures Analysis/New Failures Analysis.workbook

```diff
@@ -39,156 +39,187 @@
             "multiSelect": true,
             "quote": "'",
             "delimiter": ",",
-            "query": "requests\n| where timestamp >= ago(1d)\n| summarize Count = count() by operation_Name\n| order by Count desc\n| project v = operation_Name, t = operation_Name, s = false\n| union (datatable(v:string, t:string, s:boolean)[\n'*', 'All Operations', true\n])"
+            "query": "requests\n| where timestamp >= ago(1d)\n| summarize Count = count() by operation_Name\n| order by Count desc\n| project v = operation_Name, t = operation_Name, s = false\n| union (datatable(v:string, t:string, s:boolean)[\n'*', 'All Operations', true\n])",
+            "resourceType": "microsoft.insights/components"
           },
           {
             "id": "eac9cbf1-6364-4f28-9c61-a9b038c035d7",
             "version": "KqlParameterItem/1.0",
             "name": "OverTimeRange",
```

Comment:
are the iframes/etc that are attempting to pass time range into this workbook already using the previous string values?
(see [FailuresWorkbooksDropdown.tsx](https://msazure.visualstudio.com/One/_git/MGMT-AppInsights-Iframes?path=%2FCurated%2Fsrc%2Fcomponents%2Ffailures%2Fanalytics-dropdown%2FFailuresWorkbooksDropdown.tsx&version=GBmaster&line=123&lineEnd=124&lineStartColumn=1&lineEndColumn=1&lineStyle=plain)

you might want to rename the param if you're changing the type, then redo the code in the iframe to match?

Reply:
[Code change is here]( https://msazure.visualstudio.com/One/_git/MGMT-AppInsights-Iframes/pullrequest/3381512).